main.tf

```diff
@@ -2,15 +2,13 @@ provider "aws" {
 }
```

Comment:
go ahead and remove this provider block also. turns out it isn't needed and actually causes issue for some use cases of this module.